app/react/portainer/settings/AuthenticationView/InternalAuth/PasswordLengthSlider/PasswordLengthSlider.tsx

```diff
@@ -85,31 +83,20 @@ export function PasswordLengthSlider({
     }
   }
 
-  function handleChange(sliderValue: number) {
+  function handleChange(sliderValue: number | number[]) {
     onChange(sliderValue);
   }
 
   return (
     <div className={clsx(styles.root, styles[sliderProps.strength])}>
       <div className="col-sm-4">
-        <SliderWithTooltip
-          tipFormatter={(value) => `${value} characters`}
```

Comment:
Is '... characters' removed from this tooltip completely?

Reply:
Fixed, thanks!